### ISSUE TITLE
Include __init__.py

### ISSUE THREAD

**Issue by catboxanon** [Davemane42/ComfyUI_Dave_CustomNode#3]
Is it possible to include `__init__.py` in this repo so installing can be done via a `git clone`? Currently, if this repo is cloned into the `custom_nodes` folder, it returns:

```
Cannot import C:\AI\ComfyUI\custom_nodes\ComfyUI_Dave_CustomNode module for custom nodes: [Errno 2] No such file or directory: 'C:\\AI\\ComfyUI\\custom_nodes\\ComfyUI_Dave_CustomNode\\__init__.py'
```

For reference:
https://github.com/WASasquatch/was-node-suite-comfyui
https://github.com/hnmr293/ComfyUI-nodes-hnmr

**Comment by Davemane42** [comment]
Will do when i get home. only problem is the .py goes in custom_nodes and .js in web/extension.
ill check how it could install in the correct folders if possible

**Comment by catboxanon** [comment]
Ah, I hadn't realized the .js file goes elsewhere. If that isn't something that can be resolved that's understandable, I can symlink it, but some install documentation should be added. If you plan on adding more nodes it might make sense to separate out the JS so there's less extra manual linking to do in the future too.

**Comment by superskirv** [comment]
I would also add that a workflow.json file would be an excellent way to reduce people asking for help.

**Comment by AceroMagico** [comment]
For now is possible to not have the error message by renaming `MultiAreaConditioning.py` to `__init__.py`. see this for reference : https://github.com/WASasquatch/was-node-suite-comfyui/issues/4

**Comment by Davemane42** [comment]
fixed with [55e3bac](https://github.com/Davemane42/ComfyUI_Dave_CustomNode/commit/55e3baca3a9764362a0dc5098264b34f1da419a2)
